File: ocmsdk/src/main/java/com/gigigo/orchextra/ocm/OcmCustomBehaviourDelegate.java
package com.gigigo.orchextra.ocm;

import java.util.Map;

public interface OcmCustomBehaviourDelegate {
  void contentNeedsValidation(Map<String, String> customProperties, CompletionCallback completionCallback);

  //ViewCustomizationType customizationForContent(Map<String, String> customProperties, ViewType viewType);
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/ocm/OcmCustomBehaviourDelegate.kt
package com.gigigo.orchextra.ocm

interface OcmCustomBehaviourDelegate {
  fun contentNeedsValidation(customProperties: Map<String, Any>, completion: (Boolean) -> Unit)

  //ViewCustomizationType customizationForContent(Map<String, Any> properties, ViewType viewType);
}
